config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   namespace :api, defaults: { format: 'json' } do
     namespace :v1 do
+      resources :slides
```

Comment:
Don't need to add an addicional route:
```suggestion
      resources :slides, only: :index
```

Reply:
Please check [this](https://guides.rubyonrails.org/routing.html) to understand how to creates routes in a Ruby on Rails project.